conf/drupal/config/block.block.views_block__event_sponsors_block_5.yml

```diff
@@ -26,6 +26,6 @@ settings:
 visibility:
   request_path:
     id: request_path
-    pages: '/*/sponsors'
+    pages: /2019/sponsors
```

Comment:
This has removed the Party sponsors block from the `/2018/sponsors` page.

Reply:
I've corrected this in production. My intent was just to remove it from 2020 forward, but I forgot that 2018 was there.